src/librustc/diagnostics.rs

```diff
@@ -335,6 +335,24 @@ This error indicates that an attempt was made to divide by zero (or take the
 remainder of a zero divisor) in a static or constant expression.
 "##,
 
+E0030: r##"
+When matching against a range, the compiler verifies that the range is
+non-empty.  Range patterns include both end-points, so this is equivalent to
+requiring the start of the range to be less than or equal to the end of the
+range.
+
+For example:
+
+```
+match 5u32 {
+    // This range is ok, albeit pointless.
```

Comment:
when I first read this comment, I thought "pointless" was meant to mean "this pattern can never actually match."  But of course this is not what you meant.

Maybe phrase instead as "This range is okay, though it could be trivially replaced with just the single literal `1`" ?